charts/topolvm/templates/lvmd/psp.yaml

```
@@ -14,9 +14,13 @@ spec:
     - 'emptyDir'
     - 'hostPath'
     - 'secret'
-  {{- with .Values.lvmd.psp.allowedHostPaths }}
-  allowedHostPaths: {{ toYaml . | nindent 2 }}
-  {{- end }}
+  allowedHostPaths: 
+    {{- if .Values.lvmd.psp.allowedHostPaths }}
+    {{- toYaml .Values.lvmd.psp.allowedHostPaths | nindent 2 }}
```

```suggestion
    {{- toYaml .Values.lvmd.psp.allowedHostPaths | nindent 4 }}
```

I want to match the level of indentation with the else clause.